From b86406740d1daa395e55d58d373929cb3427ad2a Mon Sep 17 00:00:00 2001
From: skyasara <sushma.kyasaralli.thimmappa@intel.com>
Date: Tue, 11 Dec 2012 06:51:22 -0800
Subject: Dalvik: Change the JIT threshold to 50.

BZ: 75820

This patch changes the JIT threshold to 50(from 255)
Although this brings no significant boost to any of the KPIs, this is more to be
consistent with the configurations of other lead devices. Also, there is no degradation
seen with this change.

Category: device-enablement
Domain: AOSP-Dalvik-Compiler-CG
Origin: internal
Upstream-Candidate: no, needs rework

Change-Id: I51b8ecb808cc5cf3138879b50ce8df214351e822
Orig-MCG-Change-Id: Id8e69fa6e8fe0499702667e8a500eb183d68ecd3
Signed-off-by: skyasara <sushma.kyasaralli.thimmappa@intel.com>
Signed-off-by: Qiming Shi <qiming.shi@intel.com>
Signed-off-by: Serguei Katkov <serguei.i.katkov@intel.com>
---
 vm/compiler/codegen/x86/CodegenInterface.cpp |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/vm/compiler/codegen/x86/CodegenInterface.cpp b/vm/compiler/codegen/x86/CodegenInterface.cpp
index 51f879e..642439c 100644
--- a/vm/compiler/codegen/x86/CodegenInterface.cpp
+++ b/vm/compiler/codegen/x86/CodegenInterface.cpp
@@ -106,7 +106,7 @@ bool dvmIsOpcodeSupportedByJit(const DecodedInstruction & insn)
 //! default JIT table size used by x86 JIT
 #define DEFAULT_X86_ATOM_DALVIK_JIT_TABLE_SIZE 1<<12
 //! default JIT threshold used by x86 JIT
-#define DEFAULT_X86_ATOM_DALVIK_JIT_THRESHOLD 255
+#define DEFAULT_X86_ATOM_DALVIK_JIT_THRESHOLD 50
 //! default JIT code cache size used by x86 JIT
 #define DEFAULT_X86_ATOM_DALVIK_JIT_CODE_CACHE_SIZE 512*1024
 
-- 
1.7.4.1

